protocols/gossipsub/src/types.rs

```diff
@@ -514,3 +586,237 @@ impl fmt::Display for PeerKind {
         f.write_str(self.as_ref())
     }
 }
+
+/// `RpcOut` sender that is priority aware.
+#[derive(Debug, Clone)]
+pub(crate) struct RpcSender {
+    cap: usize,
```

Comment:
```suggestion
    /// Capacity of the priority channel for `Publish` messages.
    cap: usize,
```
Also maybe renamed to `priority_cap`, to avoid confusion with the `cap` parameter in `RpcSender::new`?

Reply:
yeah makes sense, thanks Elena!